src/config/dynamic-theme-fixes.config

```diff
@@ -3433,7 +3433,7 @@ drive.google.com
 INVERT
 img[src$="google_gsuite"]
 div[role="menuitem"] svg
-div[role="dialog"] div[role="button"][tabindex="0"]
+div[role="dialog"][style^="width: 350px;"] div[role="button"][tabindex="0"]
```

Comment:
Does this play well with different screen resolutions?

Reply:
Yes, I checked.

Reply:
Great. Il take a look to fix the other problem that was mentioned.